helios-services/src/main/java/com/spotify/helios/master/resources/HostsResource.java

```diff
@@ -161,12 +170,15 @@ public HostDeregisterResponse delete(@PathParam("id") final String host) {
   @Produces(APPLICATION_JSON)
   @Timed
   @ExceptionMetered
-  public Map<String, HostStatus> hostStatuses(final List<String> hosts) {
+  public Map<String, HostStatus> hostStatuses(final List<String> hosts,
+                                              @QueryParam("status") final String statusFilter) {
```

Comment:
Same thing here.

A test checking that it works to call this endpoint wihtout the status param would be nice.